viewsb/frontends/qt.py

```diff
@@ -77,7 +78,7 @@ def __init__(self):
         QtCore.QCoreApplication.setAttribute(QtCore.Qt.AA_ShareOpenGLContexts)
 
         self.app = QApplication([])
-        self.ui_file = QtCore.QFile('viewsb/frontends/qt.ui')
+        self.ui_file = QtCore.QFile(os.path.dirname(os.path.realpath(__file__)) + '/qt.ui')
```

Comment:
This isn't strictly enough -- this needs to be included in the `MANIFEST.in` in order to ensure it makes its way into packages.

Reply:
See e.g.: https://docs.python.org/3/distutils/sourcedist.html

Reply:
On second thought, this might be more appropriate as a data-file: https://docs.python.org/3/distutils/setupscript.html#distutils-additional-files